resources/js/Types/Option.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Option{
```

Comment:
Do you need export if this is a `d.ts` file ?

Reply:
Yes, it doesn't work otherwise

Reply:
Please me explain why this necessary. Give me example where this is not work and how you use this interface.

Reply:
Fixed.

Sorry, it works. Global type declarations don't get recognized when passed directly to a macro as type arguments (`defineProps<Opition>`), but here I use `Opition` as inline object type instead, so it works.

Reply:
:rocket: